### PR TITLE
fix(deps): Update dependency adbkit to v3.2.3

### PR DESCRIPTION
**Pull request by rpl**

### CODE REVIEW

File: tests/functional/test.lib.imports.js [hunk 1]
```diff
@@ -9,8 +9,56 @@ import {
...
+    process.env.HOME, '.npm-global'
+  );
+  if (!shell.test('-d', npmDir)) {
+    shell.echo(`Creating test npm directory at ${npmDir}`);
+    shell.mkdir(npmDir);
+  }
+  return npmDir;
+}
+
+function npm_link() {
```
Comment (willdurand) [inline tests/functional/test.lib.imports.js:31]:
```suggestion
function npmLink() {
```

File: tests/functional/test.lib.imports.js [hunk 2]
```diff
@@ -9,8 +9,56 @@ import {
...
+      cwd: path.resolve(path.join(__dirname, '..', '..')),
+      env: {
+        ...process.env,
+        NPM_CONFIG_PREFIX: npmGlobalDir(),
+      },
+    });
+  }
+}
+
+function npm_unlink() {
```
Comment (willdurand) [inline tests/functional/test.lib.imports.js:47]:
```suggestion
function npmUnlink() {
```

File: tests/functional/test.lib.imports.js [hunk 3]
```diff
@@ -9,8 +9,56 @@ import {
   withTempDir, fixturesUseAsLibrary,
 } from './common';
 
-const npm = shell.which('npm').toString();
-const node = shell.which('node').toString();
+const npm = shell.which('npm')?.toString();
+const node = shell.which('node')?.toString();
+const nvm = shell.which('nvm')?.toString();
+const isWindow = process.platform === 'win32';
```
Comment (willdurand) [inline tests/functional/test.lib.imports.js:15]:
```suggestion
const isWindows = process.platform === 'win32';
```

**Review by willdurand — APPROVED** [review]
r+wc but I think you should extract the CI changes in a separate PR.

**Comment by rpl** [comment]
> r+wc but I think you should extract the CI changes in a separate PR.

I'm going to merge this using "Rebase and merge" which will keep the pieces logically separated from each other as it is in the 3 commits part of the PR.

**Pull request merged**